src/modules/schulconnex/domain/personeninfo/personeninfo.service.ts

```diff
@@ -57,13 +57,20 @@ export class PersonenInfoService {
             return [];
         }
 
-        const personIds: PersonID[] =
-            await this.schulconnexRepo.findPersonIdsWithKontextAtServiceProvidersAndOptionallyOrganisations(
+        const [idsWithKontext, idsWithRollenerweiterung]: [PersonID[], PersonID[]] = await Promise.all([
+            this.schulconnexRepo.findPersonIdsWithKontextAtServiceProvidersAndOptionallyOrganisations(
                 permittedServiceProviderIds,
                 permittedOrgas.all ? 'all' : new Set<string>(permittedOrgas.orgaIds),
-                offset,
-                limit,
-            );
+            ),
+            this.schulconnexRepo.findPersonIdsWithRollenerweiterungForServiceProviderAndOptionallyOrganisations(
+                permittedServiceProviderIds,
+                permittedOrgas.all ? 'all' : new Set<string>(permittedOrgas.orgaIds),
+            ),
+        ]);
+
+        const personIds: PersonID[] = Array.from(new Set([...idsWithRollenerweiterung, ...idsWithKontext]))
+            .sort()
```

Comment:
SQ complains, because it thinks sorting alphabetically might be unintentional. It should be fine, if we provide an explicit compare-function.
```suggestion
            .sort((a: string, b: string) => a.localeCompare(b))
```